### PR TITLE
fix(transaction notification): fix transactions notification template

### PR DESCRIPTION
**Pull request by VladislavSokov**
Screenshot from 2023-12-01 19-47-49

### CODE REVIEW

**Comment by ka8725** [comment]
@VladislavSokov it's looking nice! WDYT if we highlight +100 with green and -100 with red? (Actually, IMO we should do that in the history table as well)

**Comment by VladislavSokov** [comment]
I'll try.

File: app/views/transactions_mailer/transaction_notification.html.erb [hunk 1]
```diff
@@ -1,21 +1,98 @@
...
-<table>
-  <tr>
-    <th>Date</th>
-    <th>Amount</th>
-    <th>Description</th>
-  </tr>
-  <% @transactions.each do |transaction|%>
+<div class="container">
+  <h1>Transaction added to <%= @account.name %></h1>
+  <h2>About transation:</h3>
```
Comment (ka8725) [inline app/views/transactions_mailer/transaction_notification.html.erb:3]: let's remove this line. It's clear from the context what's below

**Comment by VladislavSokov** [comment]
Screenshot from 2023-12-04 14-30-54

**Pull request merged**